File: src/main/java/com/amberclient/modules/render/NoHurtCam.java
package com.amberclient.modules.render;

import com.amberclient.utils.module.Module;

public class NoHurtCam extends Module {
    private static NoHurtCam instance;

    public NoHurtCam() {
        super("NoHurtCam", "X", "Render");
        instance = this;
    }

    public static NoHurtCam getInstance() {
        return instance;
    }
}

File: src/main/java/com/amberclient/modules/render/NoHurtCam.kt
package com.amberclient.modules.render

import com.amberclient.utils.module.Module

class NoHurtCam : Module("NoHurtCam", "Disables the camera shake effect", "Render") {

    init {
        instance = this
    }

    companion object {
        @JvmStatic
        private var instance: NoHurtCam? = null

        @JvmStatic
        fun getInstance(): NoHurtCam? {
            return instance
        }
    }
}